debian/rules

```diff
@@ -134,6 +134,14 @@ override_dh_strip:
 override_dh_shlibdeps:
```

Comment:
the commit message confuses me; if the warning as about running dh_python2 against python-rbd, the code below explicitly does that; did you mean to have a different package name in the warning above?...